### PR TITLE
Fix disabled input not being cleared if last one

### PR DESCRIPTION
**Pull request by daleeidd**
Flow, ~~clip and depth~~ simulations where not being cleared when the last input is disabled.

### CODE REVIEW

File: crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrClipSurface.cs [hunk 1]
```diff
@@ -60,6 +60,8 @@ public override void BuildCommandBuffer(OceanRenderer ocean, CommandBuffer buf)
                 return;
             }
 
+            _targetsClear = false;
```
Comment (huwb) [inline crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrClipSurface.cs:63]: Interesting.. targetsClear is anyway written below the loop, so setting it false here only impacts the loop.. I think.. Is the value used within the loop, hence setting it here fixes it? Just trying to get my head around what is going wrong.

**Review by huwb — APPROVED** [review]
Approving as it looks safe enough, but a comment on why it's needed might be useful so we (I) don't deem the extra assignment superfluous and remove it in the future.

File: crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrClipSurface.cs (hunk 1 above)
Comment (daleeidd) [reply crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrClipSurface.cs]: Apologies. I made a mistake. Only flow was the problem. I have rebased with the correct fix for flow.

File: docs/about/history.rst [hunk 2]
```diff
@@ -27,6 +27,7 @@ Fixed
    -  Fix ocean not rendering on Xbox One and Xbox Series X.
    -  Fix height input (and others) from not working 100m above sea level and 500m below sea level.
    -  Fix FFT shader build errors for Game Core platforms.
+   -  Fix flow simulation sometimes not clearing disable input.
```
Comment (huwb) [inline docs/about/history.rst:30]: Oh minor gramma .. "..not clearing after disabling last.."

**Review by huwb — APPROVED** [review]
Oh yep that's a proper logic fail! Thanks!

**Comment by huwb** [comment]
Logic fail on my side, I meant haha

**Comment by daleeidd** [comment]
All good. Sweet. Thanks for the review.

**Pull request merged**